packages/contractkit/src/wrappers/Election.ts

```diff
@@ -330,4 +372,72 @@ export class ElectionWrapper extends BaseWrapper<Election> {
       greater: newIdx === currentVotes.length - 1 ? NULL_ADDRESS : currentVotes[newIdx + 1].address,
     }
   }
+
+  /**
+   * Retrieves GroupVoterRewards at epochNumber.
+   * @param epochNumber The epoch to retrieve GroupVoterRewards at.
+   */
+  async getGroupVoterRewards(epochNumber: number): Promise<GroupVoterReward[]> {
```

Comment:
It seems to me that `validators` is not really something that belongs to `GroupVoterReward`. Seems like we are doing a lot of calls for something that not everybody would want.
I mean, we asked for the rewards for each group. Having the member of each group is not something we asked for